tests/device_tests/test_sdcard.py

```diff
@@ -113,7 +113,10 @@ def input_flow_change_pin_format():
         client.debug.input("1234")
 
         yield  # SD card problem
-        TR.assert_in(layout().text_content(), "sd_card__unplug_and_insert_correct")
+        if client.model is models.T2T1:
```

Comment:
not exactly correct as this is dependent also on the date of manufacturing. But it's true for the emulator, so let's keep it.

Reply:
Good point, 0079dc0045391b8e615caf2d162e447320a0296c now allows both variants of the text.

Reply:
I commented there, didn't you duplicate a function?